### ISSUE TITLE
Responsive layout

### ISSUE THREAD

**Issue by penzur** [bis-hack/bishack.dev#10]

**Comment by rbo13** [comment]
Hi @penzur! Do you want bootstrap gaming for this one? 😂

**Comment by jkbicbic** [comment]
I think this issue has been addressed by **PR** #14, #15, #16 @penzur :D

**Comment by penzur** [comment]
yup... but can't close yet coz the post section not yet responsive lah nyahahaha

![image](https://user-images.githubusercontent.com/191376/78472444-6a7cef00-776b-11ea-82aa-bc0d2e966000.png)

@jkbicbic @rbo13

**Comment by jkbicbic** [comment]
Oh no la, forgot about that part, tho I have an issue with creating post hmmmm 🤔will recreate the issue where I can't create a post then notify you for help @penzur

**Comment by penzur** [comment]
it's ok leh... no rush, mah! :D
